samples/matter/smoke_co_alarm/README.rst

```diff
@@ -0,0 +1,379 @@
+
+Configuration
+*************
+
+|config|
+
+.. _matter_smoke_co_alarm_custom_configs:
+
+Matter smoke CO alarm custom configurations
+===========================================
+
+.. include:: ../light_bulb/README.rst
+    :start-after: matter_light_bulb_sample_configuration_file_types_start
+    :end-before: matter_light_bulb_sample_configuration_file_types_end
+
+Device Firmware Upgrade support
+===============================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_build_with_dfu_start
+    :end-before: matter_door_lock_sample_build_with_dfu_end
+
+FEM support
+===========
+
+.. include:: /includes/sample_fem_support.txt
+
+Factory data support
+====================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_factory_data_start
+    :end-before: matter_door_lock_sample_factory_data_end
+
+User interface
+**************
+
+.. tabs::
+
+   .. group-tab:: nRF52 and nRF53 DKs
+
+      LED 1:
+         .. include:: /includes/matter_sample_state_led.txt
+
+      LED 2:
+         The following states are possible:
+
+         * Blinking evenly 300 ms on/300 ms off when the smoke alarm is expressed.
+         * Blinking evenly 500 ms on/500 ms off when the Identify command of the Identify cluster is received on the endpoint ``1``.
+           The command's argument can be used to specify the duration of the effect.
+         * Blinking in combination with LED3 and LED4 when other alarms are triggered. 
+
+      LED 3:
+         The following states are possible:
+
+         * Blinking evenly 300 ms on/300 ms off when the CO alarm is expressed.
+         * Blinking in combination with LED2 and LED4 when other alarms are triggered. 
+
+      LED 4:
+         The following states are possible:
+
+         * Blinking evenly 300 ms on/300 ms off when the battery low level alarm is expressed.
+         * Blinking in combination with LED2 and LED3 when other alarms are triggered. 
+
+      LED 2, LED 3 and LED 4 combined:
+         The following states are possible:
+
+         * All three LEDs linking at the same time oddly 300 ms on/700 ms off when the hardware fault alarm is expressed.
+         * All three LEDs linking at the same time oddly 700 ms on/300 ms off when the end of service alarm is expressed.
+         * Flushing in sequence from LED2, through LED3 to LED4 with 200 ms interval when the self-test is triggered.
+
+      Button 1:
+         .. include:: /includes/matter_sample_button.txt
+
+      Button 3:
+         Functions as the User Active Mode Trigger (UAT) button.
+         For more information about Intermittently Connected Devices (ICD) and User Active Mode Trigger, see the :ref:`ug_matter_device_low_power_icd` documentation section.
+
+      .. include:: /includes/matter_segger_usb.txt
+
+   .. group-tab:: nRF54 DKs
+
+      LED 0:
+         .. include:: /includes/matter_sample_state_led.txt
+
+      LED 1:
+         The following states are possible:
```

Comment:
The corresponding changes from the other tab apply to this one as well, of course.